office365_admin/komand_office365_admin/actions/add_user/action.py

```diff
@@ -40,7 +40,7 @@ def run(self, params={}):
             raise PluginException("There was an issue with the Add User request. Double-check the request body:", data=body)
             
         if response.status_code == 201:
-            return {Output.USER: response.json()}
+            return {Output.USERS: response.json()}
```

Comment:
I don't think that line should have changed. Make sure that lines up with the spec.

Reply:
USER

Reply:
yes this should not be here.